dashboard/test/ui/features/star_labs/applab/level_options.feature

```diff
@@ -4,6 +4,7 @@ Feature: App Lab Level Options
 Scenario: Table data in level definition appears in data browser
   Given I am on "http://studio.code.org/s/allthethings/lessons/18/levels/16"
   And I wait for the lab page to fully load
+  And I wait for 1 second
```

Comment:
Yup, I can add a comment and bump to seconds. This is another case where the step `I wait for the lab page to fully load` is misleading, because iirc that just checks that the Run button has been rendered and doesn't account for other asynchronous loading.

Reply:
Tagging @cnbrenci here as an FYI since she has lots of context in datablock storage.